.github/ISSUE_TEMPLATE/new-content.md

```diff
@@ -4,7 +4,8 @@ about: Submit content to be reviewed and published
 title: Content's title
 labels: content
 assignees: ''
-
+twitter_body: Specific body for twitter
+twitter_hashtags: ['list', 'of', 'hashtags']
```

Comment:
In order to merge this change independently, could you please rebase from current `main` branch without yet the `twitter_*` fields.

Reply:
My bad... too early means too much errors :smiley: